extensions/wikia/UserProfilePageV3/templates/UserProfilePage_WikiaMobileRenderUserIdentityBox.php

```diff
@@ -1,47 +1,46 @@
 <section class=mst>
 	<div class=mstHhdr>
-		<!-- <img src="<?= $wg->BlankImgURL ;?>" data-src="<?= $user['avatar']; ?>" itemprop=image class="lazy imgPlcHld"> --!>
-		<!-- <noscript><img src="<?= $user['avatar']; ?>" itemprop=image></noscript> -->
 		<div class=mstAva>
-			<img class=mstAvaImg src="<?= $user['avatar']; ?>" itemprop=image>
+			<img class=mstAvaImg src="<?= Sanitizer::encodeAttribute( $user['avatar'] ); ?>" itemprop=image>
 		</div>
 		<hgroup class=mstHgr>
 			<? if( !empty($user['name']) ): ?>
-				<h1 class=mstNm itemprop=name><?= $user['name']; ?></h1>
+				<h1 class=mstNm itemprop=name><?= htmlspecialchars( $user['name'] ); ?></h1>
 			<? endif; ?>
 			<? if( !empty($user['tags']) ): ?>
 				<?php foreach($user['tags'] as $tag): ?>
 					<span class=mstGr><?= $tag; ?></span>
 				<?php endforeach; ?>
 			<? endif; ?>
 			<? if(!empty($user['chatBanned'])): ?>
-				<span class=mstGr><?= wfMsg('user-identity-box-banned-from-chat'); ?></span>
+				<span class=mstGr><?= wfMessage( 'user-identity-box-banned-from-chat' )->escaped(); ?></span>
 			<? endif; ?>
 			<? if( !empty($user['realName']) ): ?>
-				<h2 class=mstRn><?= wfMsg('user-identity-box-aka-label', array('$1' => $user['realName']) ); ?></h2>
+				<h2 class=mstRn><?= wfMessage( 'user-identity-box-aka-label', htmlspecialchars( $user['realName'] ) )->plain(); ?></h2>
 			<? endif; ?>
 			<?php if( $user['edits'] >= 0 ): ?>
-				<p class=mstEd><?= wfMsg('user-identity-box-edits', array( '$1' => $user['edits'] ) ); ?></p>
+				<p class=mstEd><?= wfMessage( 'user-identity-box-edits', htmlspecialchars( $user['edits'] ) )->plain(); ?></p>
 			<?php endif; ?>
 		</hgroup>
 	</div>
 
 	<div class=mstInf>
 		<ul class=mstUl>
 			<? if( !empty($user['location']) ): ?>
-				<li class=mstSectLi itemprop=address><?= wfMsg('user-identity-box-location', array( '$1' => $user['location'] )); ?></li>
+				<li class=mstSectLi itemprop=address><?= wfMessage( 'user-identity-box-location', htmlspecialchars( $user['location'] ) )->plain(); ?></li>
 			<? endif; ?>
 			<? if( !empty($user['birthday']) && intval( $user['birthday']['month'] ) > 0 && intval( $user['birthday']['month'] ) < 13 ): ?>
-				<li class=mstSectLi><?= wfMsg('user-identity-box-was-born-on', array( '$1' => F::app()->wg->Lang->getMonthName( intval($user['birthday']['month']) ), '$2' => $user['birthday']['day'] )); ?></li>
+				<li class=mstSectLi><?= wfMessage( 'user-identity-box-was-born-on', F::app()->wg->Lang->getMonthName( intval( $user['birthday']['month'] ) ), htmlspecialchars( $user['birthday']['day'] ) )->plain();
```

Comment:
Since we're expecting `day` to be a number (and are enforcing that on the month), we could also wrap `$user['birthday']['day']` in `intval` so we know it's a number on output. What do you think? Same below.